README.md

```diff
@@ -1,10 +1,12 @@
-# Vulcanize DB
+# VDB-Lite
 
-[![Build Status](https://travis-ci.org/makerdao/vulcanizedb.svg?branch=master)](https://travis-ci.org/makerdao/vulcanizedb)
-[![Go Report Card](https://goreportcard.com/badge/github.com/makerdao/vulcanizedb)](https://goreportcard.com/report/github.com/makerdao/vulcanizedb)
+[![Build Status](https://travis-ci.com/makerdao/vdb-lite.svg?branch=staging)](https://travis-ci.com/makerdao/vdb-lite)
+[![Go Report Card](https://goreportcard.com/badge/github.com/makerdao/vdb-lite)](https://goreportcard.com/badge/github.com/makerdao/vdb-lite)
```

Comment:
Not sure if this was intentional, but this badge doesn't actually take you to the report when you click it. Maybe the second link could be replaced with https://goreportcard.com/report/github.com/makerdao/vdb-lite ?

Reply:
🤦‍♂